src/network/network.service.ts

```diff
@@ -29,6 +30,12 @@ export class NetworkService {
     return this.polymeshService.polymeshApi.network.getTreasuryBalance();
   }
 
+  public async getProtocolFees(tags: TxTag[], blockHash?: string): Promise<ProtocolFees[]> {
+    const args = (blockHash ? { tags, blockHash } : { tags }) as unknown as { tags: TxTag[] };
```

Comment:
```suggestion
    const args = (blockHash ? { tags, blockHash } : { tags }) };
```

There shouldn't be type casts like this outside of tests.